implementations/rust/ockam/ockam/src/system/hooks/mod.rs

```diff
@@ -0,0 +1,11 @@
+#![allow(unused)] // FIXME
```

Comment:
FIXME 😉

Reply:
(Oh, and this)

Reply:
woops :see_no_evil:

Reply:
It's strange that my git-grep search for "unused" didn't show this line O.o